benchmarks/light_benchmark.py

```diff
@@ -11,25 +11,29 @@
 # Modified from plot_classify_P300_bi.py of pyRiemann
 # License: BSD (3-clause)
 
-from pyriemann.estimation import XdawnCovariances, Shrinkage
-from pyriemann.tangentspace import TangentSpace
-from sklearn.pipeline import make_pipeline
-from sklearn.model_selection import train_test_split
-from sklearn.metrics import balanced_accuracy_score
-from sklearn.preprocessing import LabelEncoder
-from sklearn.discriminant_analysis import LinearDiscriminantAnalysis as LDA
-from sklearn.decomposition import PCA
 from moabb import set_log_level
 from moabb.datasets import bi2012
 from moabb.paradigms import P300
-from pyriemann_qiskit.utils import distance, mean  # noqa
+from pyriemann.estimation import XdawnCovariances, Shrinkage
+from pyriemann.tangentspace import TangentSpace
+from pyriemann_qiskit.classification import QuanticNCH
 from pyriemann_qiskit.pipelines import (
     QuantumClassifierWithDefaultRiemannianPipeline,
     QuantumMDMWithRiemannianPipeline,
 )
-from pyriemann_qiskit.classification import QuanticNCH
-import warnings
+
+from pyriemann_qiskit.utils import distance, mean  # noqa
+from pyriemann_qiskit.utils.hyper_params_factory import create_mixer_rotational_X_gates
+from qiskit_algorithms.optimizers import SPSA
+from sklearn.decomposition import PCA
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis as LDA
+from sklearn.metrics import balanced_accuracy_score
+from sklearn.model_selection import train_test_split
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import LabelEncoder
 import sys
+import warnings
```

Comment:
`sys` and `warnings` are standard libraries, and must be imported in the first block of imports.